modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -223,7 +223,7 @@ Creates a new user, or if a user with the same name exists, replace it.
 
 [NOTE]
 ====
-It is mandatory to specify at least either a `SET PASSWORD` or a `SET AUTH` clause because users must have at least one Auth Provider.
+It is mandatory to specify a `SET PASSWORD` or a `SET AUTH` clause or both because users must have at least one auth provider.
```

Comment:
```suggestion
It is mandatory to specify a `SET PASSWORD` and/or at least one `SET AUTH` clause because users must have at least one auth provider.
```